client/packages/invoices/src/InboundShipment/DetailView/SidePanel/SidePanel.tsx

```diff
@@ -36,6 +37,7 @@ export const SidePanel: FC = () => {
       <AdditionalInfoSection />
       <RelatedDocumentsSection />
       <PricingSection />
+      {isTransfer && <TransportSection />}
```

Comment:
decided that there's no point showing this for manual shipments.. the transport reference isn't editable, so will never be populated